lib/rules/max-statements-per-line.js

```diff
@@ -35,11 +35,12 @@ module.exports = {
         const sourceCode = context.getSourceCode(),
             options = context.options[0] || {},
             maxStatementsPerLine = typeof options.max !== "undefined" ? options.max : 1,
-            message = "This line has too many statements. Maximum allowed is {{maxStatementsPerLine}}.",
-            data = { maxStatementsPerLine };
+            emptyBlockMessage = "This line has an empty block, but the maximum allowed is 0.",
```

Comment:
This message doesn't seem to fit. It reads like the maximum number of empty blocks is zero, but it's actually the maximum number of statements. I'm actually a bit confused as to why this rule checks this condition at all since we have `no-empty` already. 

@mysticatea any idea? It looks like you were the last to touch this.